common-template/src/main/resources/templates/drawer/Default/instantBody.md

```diff
@@ -0,0 +1 @@
+{#if data.context.display_name??}{data.context.display_name} triggered {data.events.size()} event{#if data.events.size() > 1}s{/if}{#else}{data.events.size()} event{#if data.events.size() > 1}s{/if} triggered{/if}
```

Comment:
This template doesn't look actionable. Will it help users somehow?

Reply:
It's not really, it's more for debug/integration process like we have for emails.
It may not be available in prod (like default email template)